netmap/netmap_contract.go

```diff
@@ -95,8 +96,6 @@ func _deploy(data any, isUpdate bool) {
 
 	var args = data.(struct {
 		notaryDisabled bool
-		addrBalance    interop.Hash160
-		addrContainer  interop.Hash160
```

Comment:
So we don't care about compatibility with the old caller code here, right? The caller must know about this change and provide correct arguments, and if the caller's version is old, then deployment fails, and we're OK with it?

Reply:
not sure. that is my first contract _change_ (not extension) and that is how i would go but lets ask @roman-khimov

Reply:
https://github.com/nspcc-dev/neofs-contract/blob/beb9293103656cef87475bc2532388ec2501e56f/neofsid/neofsid_contract.go#L56

Like this. It's absolutely critically important to keep these, even if we're not using them.

Reply:
returned